Unsafe_Invalid_1.whiley:6: unsafe invocation not permitted in safe context
    assert f(1) == 1
           ^^^^